### ISSUE TITLE
Not using SpinupWP notice

### ISSUE THREAD

**Issue by A5hleyRich** [deliciousbrains/spinupwp-plugin#4]
I've added the required logic to SpinupWP so that we can identify sites that aren't hosted using SpinupWP. @bradt as per our meeting, do we want to add a notice for those users which install our plugin?

**Comment by bradt** [comment]
Yes, I think a dashboard-wide notice makes sense. Perhaps @lizDB can help you with the copy?

**Comment by lizDB** [comment]
How about:

> It looks like this site isn't hosted on SpinupWP. Before this plugin can do anything, you'll need to add this site to your SpinupWP server. Not a SpinupWP user yet? Sign up [here](https://spinupwp.com/?utm_campaign=SpinupWP&utm_source=SpinupWP%2BMU&utm_medium=insideplugin).

Could do with an appropriate doc link on for adding the site to a SpinupWP server?

(also note the tags on the "here" link)

**Comment by bradt** [comment]
I've updated the link above to be "Sign up" instead of "here"
